### DISCUSSION TITLE
conflict with gatsby-source-strapi and gatsby-source-filesystem

### DISCUSSION THREAD

**Discussion by PepeIksde** [gatsbyjs/gatsby#38092] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hello, I have a conflict with gatsby-source-strapi and gatsby-source-filesystem, here's the error, is there anyone who can help me with this problem?

 Error: The plugin "gatsby-source-strapi" created a node of a type owned by another plugin.
        The node type "File" is owned by "gatsby-source-filesystem".
        If you copy and pasted code from elsewhere, you'll need to pick a new type name
        for your new node(s).
        

### Reproduction Link

-

### Steps to Reproduce

1.
2.
3.
...
-

### Expected Result

-

### Actual Result

-

### Environment

```shell
"gatsby": "^5.9.1",
 "gatsby-source-filesystem": "^5.9.0",
 "gatsby-source-strapi": "^3.2.0",
```

### Config Flags

-

**Comment by LekoArts** [comment]
Please report this on the repository for `gatsby-source-strapi`

**Reply by PepeIksde** [reply]
This repository has been archived by the owner on Dec 27, 2022. It is now read-only.

**Reply by rroslaniec** [reply]
Here you can find the issue https://github.com/gatsby-uc/plugins/issues/426

@LekoArts is there a possibility to disable this check/mechanism manually? To me, it can affect many different plugins under the hood trying to download some files. I'm worried that there are no active maintainers to fix the problem...

**Reply by birdcoates** [reply]
I'm having the same issue (I think) with "gatsby-source-stripe".  I'm happy to go to the repository owners of gatsby-source-stripe for assistance, although I don't know that it's being actively maintained.  

Also, an old bug in Gatsby is reported here: https://github.com/gatsbyjs/gatsby/issues/2314

Is this not the same issue and can it be fixed here rather than requiring changes to an unknown quantity of other plugins?  

Would really appreciate some advice on this.  Thanks!
